src/plugins/auto/src/cumulative_compiled_model.cpp

```diff
@@ -47,7 +47,8 @@ ov::Any AutoCumuCompiledModel::get_property(const std::string& name) const {
                                                     ov::device::properties,
                                                     ov::hint::model_priority,
                                                     ov::loaded_from_cache,
-                                                    ov::intel_auto::schedule_policy};
+                                                    ov::intel_auto::schedule_policy,
+                                                    ov::enable_profiling};
```

Comment:
we can optimize by move common code path to base class, @yangwang201911 FYI, can be in another PR

Reply:
got it